hack/dockerfiles/Dockerfile.yurthub

```diff
@@ -0,0 +1,13 @@
+# multi-arch image building for yurthub
+
+FROM --platform=${BUILDPLATFORM} golang:1.17.1 as builder
+ADD . /build
+ARG TARGETOS TARGETARCH GIT_VERSION
+WORKDIR /build/
+RUN CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} GIT_VERSION=${GIT_VERSION} make build WHAT=cmd/yurthub
+
+FROM --platform=${TARGETPLATFORM} alpine:3.14
+ARG TARGETOS TARGETARCH
+RUN apk add ca-certificates bash libc6-compat iptables ip6tables && update-ca-certificates && rm /var/cache/apk/*
```

Comment:
we should also `apk add iptables ip6tables` for yurthub, when enable-dummy-if is false, it needs to set iptables for dummy interface.